stringsAssignment.js

```diff
@@ -0,0 +1,85 @@
+// 1. console.log the first character of a string.
```

Comment:
Good flexing with use of functions. Please read questions more carefully next time as some questions had wrong output as a result. Also try to do bonus questions next time as I am sure you are capable of solving them. Bonus questions are good practice.